src/utils.h

```diff
@@ -35,6 +35,18 @@ class MountNSException : public std::exception
   std::string msg_;
 };
 
+class StderrSilencer
+{
+public:
+  StderrSilencer() = default;
+  ~StderrSilencer();
+  void silence();
+
+private:
+  int old_stderr_ = -1;
+  int new_stderr_ = -1;
```

Comment:
nit: Don't think this has to be a class member, looks like it's just used as a temp var?